framework/src/modules/nft/method.ts

```diff
@@ -365,6 +365,10 @@ export class NFTMethod extends BaseMethod {
 		const userKey = userStore.getKey(owner, nftID);
 		const userData = await userStore.get(methodContext, userKey);
 
+		if (module === NFT_NOT_LOCKED) {
+			throw new Error('Cannot be locked by NFT module');
+		}
+
```

Comment:
@mjerkov @has5aan wouldn't it make sense to have this check in the begining (L332) since it can already be checked without getting any data from store ?

Reply:
Definitely. I'll update the LIP, too...

Reply:
Updated [here](https://github.com/LiskHQ/lisk-sdk/pull/8992/commits/6367f6c1dd88034b45b072489ac87cae9498c2d7)